src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -399,6 +372,7 @@ public void siControllaEmailAggiunta() {
     public boolean controlloEmailAssociata(String email) {
         try {
             By emailBy = By.xpath("//div[@data-testid = 'courtesyContacts']//div//p[contains(text(),'" + email + "')]");
+            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(emailBy));
             return true;
         } catch (TimeoutException e) {
```

Comment:
inserire messaggio di errore

Reply:
anche a riga 387

Reply:
non serve inserire un messaggio di errore, in quanto il metodo verifica se è gia presneta una email per eliminarla altrimenti continua con il flusso di inseriemtno pec